action.yml

```diff
@@ -5,19 +5,19 @@ author: 'Prefapp'
 # Define your inputs here.
 inputs:
   dispatches_file:
-    description: 'Path to the make_dispatches.yaml file of your repo'
+    description: 'Path to the config file in the triggered repo. It will be downloaded dynamically from the GITHUB_REF of the trigger event'
```

Comment:
```suggestion
    description: 'Path to the config file in the repository. It will be downloaded dynamically from the GITHUB_SHA of the trigger event'
```